frontend/lib/src/components/widgets/DataFrame/arrowUtils.ts

```diff
@@ -604,6 +604,15 @@ export function shouldUseContentWidth(
   return widthConfig?.useContent ?? false
 }
 
+/**
+ * Helper function to determine if the element is configured to use stretch height.
+ */
+export function shouldUseStretchHeight(
```

Comment:
I think the `arrowUtils` might not be the best place for the width-related utils. It's more intended to focus on everything related to the `arrow` data format. Since the methods only seem to be used within `useTabelSizer`, maybe it's better to just move them into the `useTabelSizer.ts` file.